### PR TITLE
🤙 No more non-ruleset webhooks

### PR DESCRIPTION
**Pull request by rowanseymour**
* Remove ability to create webhook actions in editor
* Flow migration to replace webhook actions without rulesets

captura de pantalla 2018-11-05 a la s 17 06 25

becomes...

captura de pantalla 2018-11-06 a la s 13 03 48

### CODE REVIEW

File: temba/flows/tests.py [hunk 1]
```diff
@@ -9653,6 +9631,56 @@ def test_ensure_current_version(self):
         self.assertEqual(5, len(flow_json["action_sets"]))
         self.assertEqual(1, len(flow_json["rule_sets"]))
 
+    @override_settings(SEND_WEBHOOKS=True)
+    def test_migrate_to_11_7(self):
+        original = self.get_flow_json("migrate_to_11_7")
+        migrated = migrate_to_version_11_7(original)
```
Comment (norkans7) [inline temba/flows/tests.py]: can we have some asserts before we migrate?

Comment (rowanseymour) [reply temba/flows/tests.py]: What sort of asserts ?

Comment (norkans7) [reply temba/flows/tests.py]: number of action sets and rulesets

Comment (rowanseymour) [reply temba/flows/tests.py]: Ah good idea!

**Pull request merged**